roles/kibana/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+kibana_package_name: kibana
+kibana_service_name: kibana
+kibana_config_dir: /etc/kibana
+kibana_config_file: '{{ kibana_config_dir }}/kibana.yml'
+kibana_config_mode: 0644
+kibana_owner: kibana
+kibana_group: kibana
+
+# "0.0.0.0" to allow public access
+# "localhost" to allow access through proxy only
+kibana_server_bind: localhost
+kibana_server_port: 5601
+kibana_server_elasticsearch_url: 'http://{{ elasticsearch_bind }}:9200'
```

Comment:
Yeah, what richm said.  A bind address isn't necessarily useful as an address that you can contact.  Also, you may very well want to deploy Kibana on a system that is not running elasticsearch, in which case (a) you would need to manually define `elasticsearch_bind` and (b) the variable name doesn't make any sense.